### ISSUE TITLE
Hyperparameters for Yolov26

### ISSUE THREAD

**Issue by alaap001** [ultralytics/ultralytics#23926]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### ❓ Questions on Loss Functions & Hyperparameter Tuning in YOLOv26

Hi team,

I have a couple of questions and some confusion regarding loss functions and hyperparameter tuning in YOLOv26.

---

### 🔹 Q1: Box Loss Gain — Default vs Tuning Range

In the official hyperparameter tuning documentation, I see the following suggested range:

```yaml
box: tune.uniform(0.02, 0.2)
```

However, in the default config file, the value is:

```yaml
box: 7.5  # (float) box loss gain
```

This seems inconsistent.

Additionally, when I run hyperparameter tuning (using genetic algorithm) starting from the default value of `7.5`, the optimized values I get are typically around `5–8`, which are far from the recommended tuning range (`≤ 0.2`) and when I reduce it to 0.2 let's say I always get 20-30% lower accuracy than I would get from a value of 5 but official docs upper limit is 0.2 so that doesn't add up.

**Questions:**

* Why is the default value (`7.5`) so far from the suggested tuning range (`0.02–0.2`)?
* Is the tuning range outdated, or does it apply to a different scaling/normalization setup?
* Should tuning ideally start from the lower range instead of the default config?

---

### 🔹 Q2: DFL Loss Behavior in YOLOv26

From what I understand, `dfl_loss` has been removed or is no longer actively used in YOLOv26 training.

However, during training, I still observe `dfl_loss` being logged and even optimized as part of the training metrics.

**Questions:**

* Is `dfl_loss` still being computed internally but not used in backpropagation?
* If it is not used, why is it still being tracked and optimized?
* Is this expected behavior or a leftover artifact from previous versions?

---

### 🔹 Environment

* `ultralytics`: 8.4.3
* `torch`: 2.7.0+cu128
* `python`: 3.10.13

**Comment by UltralyticsAssistant** [comment]
👋 Hello @alaap001, thank you for your interest in Ultralytics 🚀! This is an automated response to help get your issue triaged quickly, and an Ultralytics engineer will also assist you soon.
We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.
Since this appears to be a YOLO26 training and hyperparameter question ❓, please provide as much information as possible, including training logs, the exact tuning command/config used, and any relevant dataset details, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).
If you believe part of this is a 🐛 bug report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. This is especially helpful for questions about logged losses, optimizer behavior, or tuning ranges.
Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.
## Upgrade
Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:
```bash
pip install -U ultralytics
```
## Environments
YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>
## Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>
If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
> Why is the default value (7.5) so far from the suggested tuning range (0.02–0.2)?

It's an old range. Probably needs to be updated.

> If reg_max > 1, you get actual DFL (cross-entropy based).
If reg_max == 1 (YOLO26 default), you get L1 loss on normalized distances instead, but the variable is still called loss_dfl.

That's correct.

**Comment by Y-T-G** [comment]
@raimbekovm Can you update the docs with the range from the code?

https://github.com/ultralytics/ultralytics/blob/b10fa7be231820cea668c52af45b25d07ff65968/ultralytics/engine/tuner.py#L91

And also the ranges in RayTune need to be updated too to be consistent with the built-in native tuner:

https://github.com/ultralytics/ultralytics/blob/b10fa7be231820cea668c52af45b25d07ff65968/ultralytics/utils/tuner.py#L56
